tests/vm_test.go

```diff
@@ -169,7 +170,28 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		type vmiBuilder func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume)
 
 		newVirtualMachineInstanceWithFileDisk := func() (*v1.VirtualMachineInstance, *cdiv1.DataVolume) {
-			return tests.NewRandomVirtualMachineInstanceWithFileDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), testsuite.GetTestNamespace(nil), corev1.ReadWriteOnce)
+			sc, foundSC := libstorage.GetRWOFileSystemStorageClass()
+			Expect(foundSC).To(BeTrue(), "Filesystem storage is not present")
+
+			imageUrl := cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine)
+			dataVolume := libdv.NewDataVolume(
+				libdv.WithRegistryURLSourceAndPullMethod(imageUrl, cdiv1.RegistryPullNode),
+				libdv.WithPVC(
+					libdv.PVCWithStorageClass(sc),
+					libdv.PVCWithVolumeSize(cd.CirrosVolumeSize),
```

Comment:
I find it confusing that Alpine is using `CirrosVolumeSize`.  Please use `cd.ContainerDiskSizeBySourceURL(imageUrl)`.

Reply:
This would do the exact same thing.
`ContainerDiskSizeBySourceURL` return either `FedoraVolumeSize` or `CirrosVolumeSize`, but since we already know we are using `cd.ContainerDiskAlpine` , there is no point in calling for it in here.

Reply:
Well, you could just define a new one named `AlineVolumeSize`.

Reply:
I recommend creating another PR for such effort, one that would change the volume size across all files and not just under this one change.
It will also move away from the purpose of this PR

Reply:
I suggested to use - `cd.ContainerDiskSizeBySourceURL(imageUrl)` since it hides the logic and avoid every vm creator to remember the Alpine should use the cirros volume size.

Reply:
This was changed to AlpineVolumeSize in the recent version